.github/workflows/basic.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
     - name: Install AVH Client for Python
       run: |
-        pip install git+https://github.com/ARM-software/avhclient.git@add_endpoint_to_s3
+        pip install git+https://github.com/ARM-software/avhclient.git@main
```

Comment:
This should probably be something less "moving" such as `@v0.1` (a branch that shall only receive patch-updated)